tap_circle_ci/streams/jobs.py

```diff
@@ -69,6 +70,12 @@ def sync(self, state: Dict, schema: Dict, stream_metadata: Dict, transformer: Tr
                 for index, (workflow_id, pipeline_id) in enumerate(pipelines[start_index:], max(start_index, 1)):
                     LOGGER.info("Syncing jobs for workflow *****%s (%s/%s)", workflow_id[-4:], index, prod_len)
                     for rec in self.get_records(workflow_id):
+
+                        # The same job can be fetched multiple times as it relates to 
+                        # multiple parents i.e workflows, this creates a composite key with the parent_id
+                        comp_key = workflow_id + rec["id"]
+                        hash_encoded = comp_key.encode('utf-8')
+                        rec['_sdc_record_hash'] = sha256_encode(hash_encoded).hexdigest()
```

Comment:
Is there a ticket for this change? could you add it to the description

Reply:
[TDL-22330](https://jira.talendforge.org/browse/TDL-22330) this is the ticket for this change,  i have updated the P.R description.

Reply:
I have removed the `_sdc_record_hash` as using a composite key will solve the issue of record duplication, these are the updated primary keys for jobs stream
Previous:
```python
    key_properties = ["id","_sdc_record_hash"]
```
Updated:
```python
    key_properties = ["id","_workflow_id"]
```